packages/apps-config/src/endpoints/testing.ts

```diff
@@ -486,7 +486,7 @@ export function createTesting (t: TFunction, firstOnly: boolean, withSort: boole
       isDisabled: true, // https://github.com/polkadot-js/apps/runs/2755409009?check_suite_focus=true
```

Comment:
If the endpoint is operational, you most-probably want to remove the isDisabled flag.

Reply:
Thanks for reminding my friend